src/mixins/amalgamation-mixin.ts

```diff
@@ -200,8 +222,8 @@ export default class AmalgamationMixin extends Vue {
     // fetch state filing
     const stateFiling = await LegalServices.fetchFiling(business.businessInfo.stateFiling)
     return (
-      stateFiling.restoration.type === RestorationTypes.LIMITED ||
-      stateFiling.restoration.type === RestorationTypes.LTD_EXTEND
+      stateFiling?.restoration?.type === RestorationTypes.LIMITED ||
```

Comment:
This here was causing lots of issues. This actually fixes the issue Mihai found where historical is not properly being evaluated. It fixes these errors:
![errors](https://github.com/bcgov/business-create-ui/assets/122301442/e801dfb9-3cf2-4289-bd0d-5b6f93b5a4e0)

Reply:
Theoretically, you shouldn't need the first `?` because line 219 verifies that there is a state filing. However, the call to fetch it might fail...

In fact, you should handle the network call failure as well: on line 223, add a catch block probably like this: `.catch(() => null) // on error, return null`

Reply:
Looks code. I have no more comments about this conversation.